src/main/java/com/genesys/roberta/tokenizer/RobertaTokenizerResources.java

```diff
@@ -78,8 +78,10 @@ private Map<BiGram, Integer> loadMergesFile(@NonNull final String resourcesPath)
                     String.format("%s merges file path: [%s] was not found", RobertaTokenizerResources.class.getSimpleName(),
                             mergesPath));
 
-            List<String> lines = Files.readAllLines(mergesPath, StandardCharsets.UTF_8);
-            return IntStream.range(0, lines.size())
+            final List<String> lines = Files.readAllLines(mergesPath, StandardCharsets.UTF_8);
+            final int startIndex = getStartingIndexOfMergesFile(lines);
```

Comment:
What do you think about iterating over `lines` and removing all lines that are comments or something like that?
Then provide the filtered `lines` to the `IntStream` below just like before.

Reply:
Sounds perfect.
Doing it, thanks!